config/environments/production.rb

```diff
@@ -63,7 +63,22 @@
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
-
+  config.action_mailer.default_url_options = { host: "https://agile-plateau-3822.herokuapp.com" }
+  
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default :charset => "utf-8"
+
+  config.action_mailer.smtp_settings = {
+  address: "aspmx.l.google.com",
+  port: 25,
+  #domain: "gmail.com" #ENV["GMAIL_DOMAIN"],
+  authentication: "plain",
+  #enable_starttls_auto: true,
+  #user_name: "teminal.service74@gmail.com",
+  #password: "ys321artem"
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.